aten/src/ATen/native/native_functions.yaml

```diff
@@ -7071,6 +7071,14 @@
   dispatch:
     CPU: histogram_cpu
 
+- func: _histogramdd_bin_edges(Tensor self, int[] bins, *, float[]? range=None, Tensor? weight=None, bool density=False) -> Tensor[]
+  dispatch:
+    CPU: histogramdd_bin_edges_cpu
+
+- func: _histogramdd(Tensor self, Tensor[] bins, *, Tensor? weight=None, bool density=False) -> Tensor
+  dispatch:
+    CPU: histogramdd_cpu
```

Comment:
Is there a plan for CUDA?

(Looks like this can't be a structured kernel because of the optional tensor, but think about providing `out=`)